source/write/insert.txt

```diff
@@ -0,0 +1,203 @@
+
+.. note::
+
+   Duplicate ``_id`` values violate unique index constraints, which
+   causes the driver to return an error.
+
+To learn more about the ``_id`` field, see the
+:manual:`Unique Indexes </core/index-unique/>` guide in the {+mdb-server+} manual.
+
+To learn more about document structure and rules, see the
+:manual:`Documents </core/document>` guide in the {+mdb-server+} manual.
+
+Insert One Document
+-------------------
+
+To add a single document to a MongoDB collection, call the ``insertOne()``
+method and pass the document you want to insert.
+
+The following example inserts a document into the ``restaurants`` collection:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/write/insert.scala
+      :language: scala
+      :start-after: start-insert-one
+      :end-before: end-insert-one
+      :dedent:
+
+   .. output::
+      :language: console
+      :visible: false
+
+      AcknowledgedInsertOneResult{insertedId=BsonObjectId{value=...}}
+      Completed
+
+Insert Multiple Documents
+-------------------------
+
+To add multiple documents to a MongoDB collection, call the ``insertMany()``
+function and pass a list of documents you want to insert.
+
+The following example inserts two documents into the ``restaurants`` collection:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/write/insert.scala
+      :language: scala
+      :start-after: start-insert-many
+      :end-before: end-insert-many
+      :dedent:
+
+   .. output::
+      :language: console
+      :visible: false
+
+      AcknowledgedInsertManyResult{insertedIds={0=BsonObjectId{value=...}, 1=BsonObjectId{value=...}}}
+      Completed
+
+Modify Insert Behavior
+----------------------
+
+The ``insertOne()`` method optionally accepts an ``InsertOneOptions``
+parameter that sets options to configure the insert operation.
+If you don't specify any options, the driver performs the insert
+operation with default settings. Pass options as the last parameter to
+the ``insertOne()`` method.
+
+The following table describes the setter methods that you can use to
+configure an ``InsertOneOptions`` instance:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``bypassDocumentValidation()``
+     - | If set to ``true``, allows the driver to ignore
+         :manual:`document-level validation </core/schema-validation>`. 
+       | Defaults to ``false``.
+
+   * - ``comment()``
+     - | Sets a comment to attach to the operation. For more information, see the :manual:`insert command
+         fields </reference/command/insert/#command-fields>` guide in the
+         {+mdb-server+} manual for more information.
+
+You can set the preceding settings on the ``insertMany()`` method
+by configuring an ``InsertManyOptions`` instance. You can also use the
+``ordered()`` method setter method to specify the order in which the driver
+inserts documents into MongoDB:
+
+.. list-table::
```

Comment:
S: since this is just one method, I don't think you need a table. Maybe just describe the ordered() method in the preceding paragraph